arch/arm/arm_features.c

```
@@ -258,23 +241,33 @@ static inline int arm_has_simd(void) {
 #define QUALCOMM_PART_ORYON 0x001
 #endif
 
+static inline int arm_has_cpuid(void) {
```

Cppcheck (2.19.0)

[style] The function 'arm_has_cpuid' is never used.

(unusedFunction)